### PR TITLE
Updated the DNS configuration

### PR DESCRIPTION
**Pull request by Th0masL**
- Updated the DNS configuration to reflect the differences between apex and sub-domains

### CODE REVIEW

File: guides/production.md [hunk 1]
```diff
@@ -25,7 +25,40 @@ On the "Networking" tab for your environment you can find the DNS and whitelisti
 
 ### DNS
 
-In order to configure your DNS provider to direct traffic for a particular set of domains to the Moovweb XDN, create a CNAME record with the value shown under _DNS Configuration_ (see above). If you are launching a brand new site then you can set this up whenever you feel ready. For sites that are already live, the DNS update is the last step. Once you have updated your DNS you are committed to launching.
+In order to configure your DNS provider to direct traffic for a particular set of domains to the Moovweb XDN, you will have to create DNS records with values depending on the type of domain you are using for your website. If you are launching a brand new site then you can set this up whenever you feel ready. For sites that are already live, the DNS update is the last step. Once you have updated your DNS you are committed to launching.
+
+Setup 1 - If you are only using a sub-domain (i.e. www.mywebsite.xyz):
```
Comment (markbrocato) [inline guides/production.md:30]:
```suggestion
# Using a sub-domain (i.e. www.mywebsite.xyz):
```

File: guides/production.md [hunk 2]
```diff
@@ -25,7 +25,40 @@ On the "Networking" tab for your environment you can find the DNS and whitelisti
 
 ### DNS
 
-In order to configure your DNS provider to direct traffic for a particular set of domains to the Moovweb XDN, create a CNAME record with the value shown under _DNS Configuration_ (see above). If you are launching a brand new site then you can set this up whenever you feel ready. For sites that are already live, the DNS update is the last step. Once you have updated your DNS you are committed to launching.
+In order to configure your DNS provider to direct traffic for a particular set of domains to the Moovweb XDN, you will have to create DNS records with values depending on the type of domain you are using for your website. If you are launching a brand new site then you can set this up whenever you feel ready. For sites that are already live, the DNS update is the last step. Once you have updated your DNS you are committed to launching.
+
+# Using a sub-domain (i.e. www.mywebsite.xyz):
+- Create a CNAME record with the value shown under _DNS Configuration_ (see above).
```
Comment (markbrocato) [inline guides/production.md:31]:
```suggestion

To host your site on a subdomain, add a CNAME record with the value shown under _DNS Configuration_ (see above).
```

File: guides/production.md (hunk 1 above)
Comment (Th0masL) [reply guides/production.md]: I guess we also want to use # for the 2 other Setup lines below, right ?

File: guides/production.md [hunk 3]
```diff
@@ -25,7 +25,41 @@ On the "Networking" tab for your environment you can find the DNS and whitelisti
...
+To host your site on a subdomain, add a CNAME record with the value shown under _DNS Configuration_ (see above).
+
+    ```DNS Setup 1
+    ; <<>> DiG 9.10.6 <<>> www.mywebsite.xyz
+    ;; ANSWER SECTION:
+    www.mywebsite.xyz.   599    IN    CNAME    d12ea738-71b3-25e8-c771-6fdd3f6bd8ba.moovweb-edge.io.
+    ```
+Setup 2 - If you are only using an apex domain (i.e. mywebsite.xyz):
+- Create multiple A records on your apex domain, with the following Anycast IP Addresses values: 151.101.1.79, 151.101.65.79, 151.101.129.79, 151.101.193.79.
+
```
Comment (markbrocato) [inline guides/production.md:41]:
```suggestion
Using an apex domain (i.e. mywebsite.xyz):

To host your site on the apex domain, create multiple `A` records on your apex domain, with the following Anycast IP Addresses values: 151.101.1.79, 151.101.65.79, 151.101.129.79, 151.101.193.79.

```

File: guides/production.md [hunk 4]
```diff
@@ -25,7 +25,42 @@ On the "Networking" tab for your environment you can find the DNS and whitelisti
...
+# Using a sub-domain (i.e. www.mywebsite.xyz):
+
+To host your site on a subdomain, add a CNAME record with the value shown under _DNS Configuration_ (see above).
+
+    ```DNS Setup 1
+    ; <<>> DiG 9.10.6 <<>> www.mywebsite.xyz
+    ;; ANSWER SECTION:
+    www.mywebsite.xyz.   599    IN    CNAME    d12ea738-71b3-25e8-c771-6fdd3f6bd8ba.moovweb-edge.io.
+    ```
+Using an apex domain (i.e. mywebsite.xyz):
```
Comment (markbrocato) [inline guides/production.md:39]:
```suggestion
# Using an apex domain (i.e. mywebsite.xyz):
```

File: guides/production.md [hunk 5]
```diff
@@ -25,7 +25,42 @@ On the "Networking" tab for your environment you can find the DNS and whitelisti
...
+    ; <<>> DiG 9.10.6 <<>> mywebsite.xyz
+    ;; ANSWER SECTION:
+    mywebsite.xyz.        599    IN    A        151.101.1.79
+    mywebsite.xyz.        599    IN    A        151.101.65.79
+    mywebsite.xyz.        599    IN    A        151.101.129.79
+    mywebsite.xyz.        599    IN    A        151.101.193.79
+    ```
+Setup 3 - If you are using both an apex domain and a sub-domain (i.e. mywebsite.xyz and www.mywebsite.xyz):
+- Create the multiple A records with the IPs, on your apex domain (see above).
+- Create a CNAME record for your sub-domain, with the value of your apex domain.
```
Comment (markbrocato) [inline guides/production.md:53]:
```suggestion
# Using both an apex domain and a sub-domain (i.e. mywebsite.xyz and www.mywebsite.xyz):

- Create the multiple A records with the IPs, on your apex domain (see above).
- Create a CNAME record for your sub-domain, with the value of your apex domain.
```

File: guides/production.md [hunk 6]
```diff
@@ -25,7 +25,42 @@ On the "Networking" tab for your environment you can find the DNS and whitelisti
...
+
+    ```DNS Setup 3
```
Comment (markbrocato) [inline guides/production.md:55]:
```suggestion
    ```
```

File: guides/production.md [hunk 7]
```diff
@@ -25,7 +25,42 @@ On the "Networking" tab for your environment you can find the DNS and whitelisti
...
+    ```DNS Setup 1
+    ; <<>> DiG 9.10.6 <<>> www.mywebsite.xyz
+    ;; ANSWER SECTION:
+    www.mywebsite.xyz.   599    IN    CNAME    d12ea738-71b3-25e8-c771-6fdd3f6bd8ba.moovweb-edge.io.
+    ```
+# Using an apex domain (i.e. mywebsite.xyz):
+
+To host your site on the apex domain, create multiple `A` records on your apex domain, with the following Anycast IP Addresses values: 151.101.1.79, 151.101.65.79, 151.101.129.79, 151.101.193.79.
+
+    ```DNS Setup 2
```
Comment (markbrocato) [inline guides/production.md:43]:
```suggestion
    ```
```

File: guides/production.md [hunk 8]
```diff
@@ -25,7 +25,42 @@ On the "Networking" tab for your environment you can find the DNS and whitelisti
...
 ### DNS
 
-In order to configure your DNS provider to direct traffic for a particular set of domains to the Moovweb XDN, create a CNAME record with the value shown under _DNS Configuration_ (see above). If you are launching a brand new site then you can set this up whenever you feel ready. For sites that are already live, the DNS update is the last step. Once you have updated your DNS you are committed to launching.
+In order to configure your DNS provider to direct traffic for a particular set of domains to the Moovweb XDN, you will have to create DNS records with values depending on the type of domain you are using for your website. If you are launching a brand new site then you can set this up whenever you feel ready. For sites that are already live, the DNS update is the last step. Once you have updated your DNS you are committed to launching.
+
+# Using a sub-domain (i.e. www.mywebsite.xyz):
+
+To host your site on a subdomain, add a CNAME record with the value shown under _DNS Configuration_ (see above).
+
+    ```DNS Setup 1
```
Comment (markbrocato) [inline guides/production.md:34]:
```suggestion
    ```
```

**Review by markbrocato — APPROVED** [review]

**Pull request merged**